mode/src/processing/mode/android/RedirectStreamHandler.java

```diff
@@ -0,0 +1,38 @@
+package processing.mode.android;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+public class RedirectStreamHandler extends Thread {
+    // Streams to redirect from and to
+    private final InputStream input;
+    private final PrintWriter output;
+
+    RedirectStreamHandler(PrintWriter output, InputStream input) {
+      this.input = input;
+      this.output = output;
+      start();
+    }
+
+    @Override
+    public void run() {
+      try {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(input));
+        String line;
+        while ((line = reader.readLine()) != null) {
+            output.println(line);
+        }
+      } catch (IOException ioe) {
```

Comment:
rename ioe to exception or something with better meaning !

please dont use abbreviations for variable names :)

Reply:
Yes, it's important to use proper variable names, I will do that. Also, I'm planning to remove this temporary stream handler file for testing and use PDE's console to print the stream.